cmd/defaults.yaml

```diff
@@ -14,6 +14,23 @@ Tracing:
   Fraction: 1.0
   MetricPrefix: zitadel
 
+Telemetry:
```

Comment:
i think there should be a mechanism to disable it completly

Reply:
You mean the milestones projection?
If we do that, this would have two major downsides, imo:
- we won't be able to query the milestones for the console
- we won't be able to push reached milestone from the past when telemetry is enabled